python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -164,15 +169,24 @@ def __really_writeIndex(self):
 
         raw_self = stripProxy(self)
 
+        import os
+
         all_caches = {}
-        for i in range(len(raw_self)):
-            this_cache = raw_self._registry.getIndexCache( stripProxy(raw_self.__getitem__(i)) )
-            all_caches[i] = this_cache
-            disk_location = raw_self.__get_dataFile(i)
-            import os
-            all_caches[i]['modified'] = os.stat(disk_location).st_ctime
+        for sj_id in range(len(raw_self)):
```

Comment:
This should really be:
```python
for sj_id, sj in enumerate(raw_self):
    ...
```
and then any `raw_self.__getitem__(sj_id)` could become just `sj`?